testing/munin/munin-node.post-install

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# create directories
+mkdir -p /usr/share/munin/docs
```

Comment:
Why do you create `/usr/share/munin/docs` here? If this package installs some docs, then it must be in APKBUILD. If it doesn’t then it doesn’t make sense to create this directory.